vendor/bat-native-ads/src/bat/ads/internal/database/tables/conversion_queue_database_table.cc

```diff
@@ -96,21 +102,20 @@ void ConversionQueue::Save(
       std::bind(&OnResultCallback, std::placeholders::_1, callback));
 }
 
-void ConversionQueue::Delete(
```

Comment:
Now database table items are not deleted. How the database will be cleaned?

Reply:
At this time the database is not cleaned. I will raise a new issue to purge items